website/frontend/templates/docs/plugin-api.html

```diff
@@ -71,6 +75,31 @@ <h1 id="file_formats">File Formats</h1>
 
       register_format(MyFile)</pre>
 
+	  <p>
+		Variables explanation:
+		<li>"PLUGIN_DESCRIPTION" should be as simple as possible, while still describing the main function</li> 
+		<li>"PLUGIN_VERSION" should be filled with the version of Plugin</li>
+		<li>"PLUGIN_API_VERSIONS" should be set to the version of Picard this plugin to run with.</li>
+		<li>"PLUGIN_LICENSE" should be set with the license name of the plugin</li> 
+		<li>"PLUGIN_LICENSE_URL" should be set with the license url</li>
+		<br>
+		<br>
```

Comment:
If you need to use 2 `<br>`s, consider maybe making it into two paragraphs (`<p>`s) instead.